auth-web/src/components/auth/BCSCPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-container id="bcsc-container" class="my-10">
+    <h2 class="mt-5">Create an Account to Incorporate</h2>
+    <p class="my-5">Log in securely using your mobile BC Services Card, government's trust way to access online services</p>
+    <v-row>
+      <!-- Image Column -->
+      <v-col cols="12" md="5">
+        <v-img src="../../assets/img/BCSC-Helper.png" aspect-ratio="1.2" contain></v-img>
+      </v-col>
+      <v-spacer></v-spacer>
+      <!-- Info Column -->
+      <v-col cols="12" md="6" class="bcsc-info-col">
+        <h2>It's Secure</h2>
+        <v-list-item class="list-item" v-for="(item, index) in secureBulletPoints" :key="index" >
+          <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <h2>It's Quick and Easy</h2>
+        <v-list-item class="list-item" v-for="(item, index) in easeBulletPoints" :key="index" >
+          <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <template v-if="!userProfile">
+          <v-btn large color="#fcba19" @click="emitLogin()" class="mt-5">
+            Log in with BC Services Card
+          </v-btn>
+          <p class="my-5">New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
+            <u>Create a BC Registries Account</u></a>
+          </p>
+        </template>
+        <LearnMoreButton />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
+
+@Component({
+  components: {
+    LearnMoreButton
+  }
+})
+export default class BCSCPanel extends Vue {
+  private secureBulletPoints: Array<any> = [
+    { text: 'A mobile card is a representation of your BC Services Card on your mobile device. It\'s used to prove who you are when you log in to access government services online.' },
+    { text: 'Only your name and a unique identifier is stored on the mobile device.' }
+  ]
+
+  private easeBulletPoints: Array<any> = [
+    { text: 'It normally takes about 5 minutes to set up a mobile card.' },
```

Comment:
Lies! I started my setup in Aug 2019 and I still haven't finished it 😛 

(Ignore this comment, obviously.)